config/routes.rb

```diff
@@ -178,6 +178,13 @@
   end
   get '/jobs', to: redirect('/')
 
+  # UofA ERA routes - footer
+  get 'policies' => 'pages#policies', id: 'policies_page'
```

Comment:
You don't need the `id: 'policies_page'` part? These make the `params[:id]` value to equal `policies_page` which isn't being used anywhere.


Also instead of hash rockets (`=>`) use the form:
```get '/contact', to: 'page#contact'```
instead